src/lib/CollectStore.js

```diff
@@ -63,9 +63,22 @@ export default class CollectStore {
   fetchAllAccounts () {
     return accounts.getAllAccounts(cozy.client, this.accountsIndex)
       .then(accounts => {
+        return Promise.all([accounts, konnectors.getAllErrors(cozy.client)])
+      })
+      .then((result) => {
+        const [accounts, errors] = result
+        const errorIndex = errors.reduce((memo, error) => {
+          memo[error.account] = error
+          return memo
+        }, {})
+
         let accObject = {}
         accounts.forEach(a => {
           if (!accObject[a.account_type]) accObject[a.account_type] = []
+          if (errorIndex[a._id]) {
+            a.isInError = true
```

Comment:
We can name it `isErrored` to keep compliant with the `errored` props used in components, what do you think?